### ISSUE TITLE
Support for Test Coverage Visualization in GitLab

### ISSUE THREAD

**Issue by SlMaker** [VSoftTechnologies/DUnitX#278]
Hello,
since DUnitX supports the JUnit XML logger (#258, #273) already it would also be nice to have support for test coverage visualization. :smile: 
https://docs.gitlab.com/ee/user/project/merge_requests/test_coverage_visualization.html
https://cobertura.github.io/cobertura/

**Comment by PsychoNineSix** [comment]
Perhaps source from [this code coverage project](https://github.com/DelphiCodeCoverage/DelphiCodeCoverage)

**Comment by vincentparrett** [comment]
I don't have plans for adding code coverage to DUnitX right now.. far too busy with other projects and my day job.

**Comment by luebbe** [comment]
>     * Fork [this code coverage project](https://github.com/DelphiCodeCoverage/DelphiCodeCoverage) and create a new output format there.

I used this code coverage project for a while using the default output format. I works fine out of the box, but it slowed down the unit tests by a factor of ten. If you run coverage tests only once a night, it won't hurt you.

**Comment by vincentparrett** [comment]
The reason it slows down the tests is because to do code coverage, you literally have to make every line a breakpoint and the coverage tool is a debugger. Think single stepping every line of code.

**Comment by luebbe** [comment]
I know, I just wanted to let the OP know what to expect.
